source/reverse-sync.txt

```diff
@@ -0,0 +1,50 @@
+.. _c2c-reverse-process:
+
+======================
+Reverse Sync Direction
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :class: singlecol
+   :depth: 1
+
+To reverse the direction of a sync operation, use the following
```

Comment:
it'd be helpful to add a note on why one might use reverse. E.g., the note we have here https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/mongosync/mongosync-behavior/#considerations-for-continuous-sync